static-site/gatsby-config.js

```diff
@@ -1,4 +1,6 @@
+const createProxyMiddleware = require("http-proxy-middleware");
```

Comment:
That was it! Commit f5f67bc added `http-proxy-middleware` as an explicit dependency, rather than a transitive dependency. This had the side-effect of upgrading the package from v0 to v1, the latter of which no longer exposes the `createProxyMiddleware` function as default, thus we need to use an explicit require to import it.